src/db.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Tsintsir'
+
+from binascii import b2a_hex
+from os import urandom
+from pymongo import MongoClient
+from pbkdf2 import PBKDF2
+
+
+def create_user(username, email, password):
```

Comment:
What happens if we call the following:

```
create_user("sameuser", "sameuser@somewhere.com", "password")
create_user("sameuser", "sameuser@somewhere.com", "password")
```

What about:
```
create_user("diffuser1", "sameuser@somewhere.com", "password")
create_user("diffuser2", "sameuser@somewhere.com", "password")
```

Perhaps we should check for and define exceptions for these cases?